server/src/main/java/com/cloud/api/query/dao/ProjectJoinDaoImpl.java

```diff
@@ -91,9 +94,11 @@ public ProjectResponse newProjectResponse(ProjectJoinVO proj) {
 
         //set resource limit/count information for the project (by getting the info of the project's account)
         Account account = _accountDao.findByIdIncludingRemoved(proj.getProjectAccountId());
-        AccountJoinVO accountJn = ApiDBUtils.newAccountView(account);
-        _accountJoinDao.setResourceLimits(accountJn, false, response);
-        response.setProjectAccountName(accountJn.getAccountName());
+        if (details.contains(DomainDetails.all) || details.contains(DomainDetails.resource)) {
+            AccountJoinVO accountJn = ApiDBUtils.newAccountView(account);
```

Comment:
line 98 takes around 0.2 seconds for each project in my testing environment.
The actual time depends on how many accounts in your system.
```
AccountJoinVO accountJn = ApiDBUtils.newAccountView(account);
```